### DISCUSSION TITLE
Blog plugin - H1 titles in archive headings

### DISCUSSION THREAD

**Discussion by seankearon** [squidfunk/mkdocs-material#7659] [Q&A] [answered]
For the blog plugin, my site generates H2 headings for the paginated archive sections:

![image](https://github.com/user-attachments/assets/c9cd7094-1481-49ea-ac50-67f9dde47065)

How can I make these H1, please?

**Comment by alexvoss** [comment]
What heading level do you have in the `blog/index.md`?

**Reply by seankearon** [reply]
Yeah, that's `H2` also!

**Reply by alexvoss** [reply]
You mean `blog/index.html` has H1 or you have a level 2 heading in `blog/index.md`? Changing that should make it H1 in the output.

**Comment by kamilkrzyskow** [comment, marked as answer]
The Blog section on the theme's website uses H1 tags properly, and testing the `blog-basic` example also works, so you had to misconfigure your website somehow. Perhaps you've set the `baselevel` of the `toc` extension to 2? <https://python-markdown.github.io/extensions/toc/>

If not you'd have to prepare a reproduction:
- https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/

**Reply by seankearon** [reply]
Thanks, @kamilkrzyskow, I did indeed have `baselevel` set to 2.  Many thanks!
